logoapp/sp/js/sp_line_data.js

```diff
@@ -546,6 +546,51 @@ LineAppFramework.prototype.onClickUploadButton = function() {
 		URL.revokeObjectURL(url);
 	};
 	*/
+	clearInterval(this.completeLoopInterval	);
+	var uploadData = new Array(8);
+	for(var i=0; i<uploadData.length; i++) {

```

Comment:
__[jshint]__
*  'i' is already defined.


<!-- Comment by [SideCI](https://sideci.com) -->